redcap_bridge/test_redcap/testfiles/elabConversion/elabFinal.json

```diff
@@ -40,20 +38,32 @@
                 "ECoG",
                 "SEEG"
             ],
+            "description": "",
             "allow_multi_values": true,
-            "position": 5
+            "position": 4
         },
         "Stimulation": {
             "type": "radio",
             "value": "",
             "options": [
                 "yes"
             ],
-            "position": 6
+            "description": "Does the recording involve stimulation?",
+            "position": 5
         },
         "Weight": {
             "type": "number",
             "value": "",
+            "description": "in kg",
+            "position": 6
+        },
+        "Immuno?": {
+            "type": "radio",
+            "value": "",
+            "options": [
+                "yes immono_custom"
```

Comment:
I think that's a typo here `immono` should be `immuno`, right?

Reply:
Yes I think the csv file contains a typo